### DISCUSSION TITLE
Godaddy Wordpress Site and plugin quit working from messaging site to production.

### DISCUSSION THREAD

**Discussion by gdharmon** [code4recovery/12-step-meeting-list#1264] [Q&A] [answered]
I successfully built out a meeting schedule using Legacy UI instead of TMSL UI on my WordPress staging site using Wordpress version 6.4.1. And 12 Step Meeting List plugin. 
After I migrated my Wordpress staging site (WP VER 6.3.2) to production in Managed Wordpress at Godaddy, the 12 Step Meeting List is not displayed on Meeting page as it had been in staging. I tried Edge Chrome, Samsung Internet  and Firefox in both tablet, smartphone and Windows.
I deactivated 12 Step Meeting List plugin and then deleted it. After I reinstalled the pkugin, the same issue although the meeting in PLuglin are still there.
I checked with Godaddy support to see if I could upgrade Wordpress version and they said no.
I already flushed Wordpress cache cleared browser cache. 
2 questIons. Is there a shortcode I can insert in a new page to bring up the meeting List?
Secondly. Second, has anyone else had this issue or know the solution?
Siter url is https//slaa-memphis.org

**Comment by joshreisner** [comment, marked as answer]
hi we see this issue when sites are migrated, the solution usually is to go to settings > permalinks and click save changes (even if you didnt make any changes) you should only have to do this once

**Reply by gdharmon** [reply]
Ok. I'll try that. THANKS.
